{"Launchpad":"https://bugs.launchpad.net/bugs/1339197","Id":"1339197","Gerrit":"https://review.openstack.org/141899","IdGerrit":"141899","IdCommit":"6ccd411831183df5eb23e14393fb890ac103929e","Files":["neutron/cmd/sanity/checks.py","neutron/cmd/sanity_check.py","neutron/plugins/linuxbridge/agent/linuxbridge_neutron_agent.py","neutron/tests/functional/sanity/test_sanity.py","neutron/tests/unit/linuxbridge/test_lb_neutron_agent.py"],"Lines_Inseted":29,"Lines_Deleted":16,"CommitParent":"0a46f3633ea13fd135d0229574b6817b348951eb","Classification":"Bug","KeywordsTitle":"requires, test","Description":"requires, to be availabe, leads to, false, negative, not mounted, could not open,ERROR, expect behaviour, current behaviour","KeywordsCommit":"moving, checks, testing","Comments":"It's a bug report"}